### ISSUE TITLE
syntax without whitespaces is not supported

### ISSUE THREAD

**Issue by tmcw** [component/is-module#3]
Test example:

```js
export{c as luminance,o as rgb,s as hex,f as score};
```

This fails the isModule check, whereas a single export, like:

```js
export{c as luminance};
```

Is correctly flagged as module.

**Comment by natevw** [comment]
Good catch! I think this is the actual cause of https://github.com/rollup/rollup-plugin-node-resolve/issues/200.

**Comment by timaschew** [comment]
> `export{c as luminance,o as rgb,s as hex,f as score};`
>
> `export{c as luminance};`

Actually none of the both examples works for me, both are returning false because of a missing whitespace between `export` and `{`

Anyway I've updated the pattern and added some tests. Now it's working.

Unfortunately I don't have permissions on npm (yet) to publish a new version.

@clintwood @jonathanong @trevorgerhardt please can anyone of you give me permissions :)

**Comment by timaschew** [comment]
ping @jonathanong @trevorgerhardt

**Comment by natevw** [comment]
Not sure if this is still the same issue anymore, but would looking for [`\b`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#special-word-boundary) instead of `\s` at least help match the actual JS grammar here?

Note the caution in MDN doc link:

> Accented characters, such as "é" or "ü" are, unfortunately, treated as non-word characters for the purposes of word boundaries, as are ideographic characters in general.
